test/lib.reviews.js

```diff
@@ -8,6 +8,7 @@ function assertValid (review) {
   assert.isString(review.id);
   assert(review.id);
   assert.isString(review.userName);
+  assert('userImage' in review);
   assert(review.userName);
```

Comment:
Are you fairly comment that there will always be a URL? I wrote the getter code defensively. Didn't want to cause flakiness in the tests but I guess it would be a good signal...let me know!